### DISCUSSION TITLE
支持转发到微信助手的微信文章、微信图片、微信小视频（含视频号视频、小程序视频、小视频文件等）

### DISCUSSION THREAD

**Discussion by cjterrence** [Quorafind/Obsidian-Thino#580] [Polls 投票]
### Feature Requested | 请求的功能

1 实现方面：第一步可以先实现发到微信助手的微信文章、微信图片，可以显示其相应的链接及文件名称到thino及obsidian。
第二步，再实现微信小视频（含视频号视频、小程序视频、小视频文件等）
2 功能方面：就是有个跳转链接就行，不用下载视频到obsidian；

### Checklist | 检查清单

- [X] The feature would be useful to more users than just me. 该功能对于不仅仅是我自己的用户来说是有用的。

[a comment by an automated account is not shown]

**Comment by juestchaos** [comment]
需要此功能的用户，可以在这里投票留言

**Reply by Ryoga1016** [reply]
需要此功能的。

**Reply by lj723911** [reply]
需要此功能，最好不要是在在公众号回复，因为微信里无法实现讲文章、对话等内容转发给公众号【PKmer】。通过企业微信的方式，支持用户给企业微信发信息，然后信息会同步保存到thino。

这个流程有些笔记类产品是已经实现的，可以参考“wolai”。

**Reply by GuanLdong** [reply]
需要该功能。

[a comment by an automated account is not shown]
